worker/executor.go

```diff
@@ -185,8 +185,8 @@ func (e *executor) worker(mut *mutation) {
 
 	// Decrease inDeg of dependents. If this mutation unblocks them, queue them.
 	for _, dependent := range mut.dependentMutations {
-		dependent.inDeg -= 1
-		if dependent.inDeg == 0 {
+		atomic.AddInt64(&dependent.inDeg, -1)
+		if atomic.LoadInt64(&dependent.inDeg) == 0 {
```

Comment:
```suggestion
		
		if atomic.AddInt64(&dependent.inDeg, -1) == 0 {
```

Reply:
It has been running fine since last night. I think this will fix the crash. I'll let it run for a day or two.